src/test/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSourceDeserializationTest.java

```diff
@@ -0,0 +1,56 @@
+package io.jenkins.plugins.gitlabbranchsource;
+
+import java.lang.reflect.Field;
+import jenkins.branch.BranchSource;
+import jenkins.scm.api.SCMSource;
+import org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import static org.junit.Assert.assertNotNull;
+
+public class GitLabSCMSourceDeserializationTest {
+
+    private static final String PROJECT_NAME = "project";
+    private static final String SOURCE_ID = "id";
+
+    @Rule
+    public final RestartableJenkinsRule plan = new RestartableJenkinsRule();
+
+    @Test
+    public void afterRestartingJenkinsTransientFieldsAreNotNull() throws Exception {
+        plan.addStep(new Statement() {
+
+            @Override
+            public void evaluate() throws Throwable {
+                GitLabSCMSourceBuilder sb = new GitLabSCMSourceBuilder(SOURCE_ID, "server", "creds", "po", "group/project");
+                WorkflowMultiBranchProject project = plan.j.createProject(WorkflowMultiBranchProject.class, PROJECT_NAME);
```

Comment:
Inside the lambda with the suggestion above you can use `j` directly from the lambda
```suggestion
                WorkflowMultiBranchProject project = j.createProject(WorkflowMultiBranchProject.class, PROJECT_NAME);
```

Reply:
Yeah! That's more convenient. Thank you. Pushed updated test.